internal/driver/config/namespace_memory.go

```diff
@@ -26,14 +26,24 @@ func NewMemoryNamespaceManager(nn ...*namespace.Namespace) *memoryNamespaceManag
 	return &nm
 }
 
-func (s *memoryNamespaceManager) GetNamespace(_ context.Context, name string) (*namespace.Namespace, error) {
+func (s *memoryNamespaceManager) GetNamespaceByName(_ context.Context, name string) (*namespace.Namespace, error) {
 	for _, n := range *s {
 		if n.Name == name {
 			return n, nil
 		}
 	}
 
-	return nil, errors.WithStack(herodot.ErrNotFound.WithReason("unknown namespace " + name))
+	return nil, errors.WithStack(herodot.ErrNotFound.WithReason("unknown namespace with name " + name))
```

Comment:
Reasons should start with capitals and end with punctuation and use `'` :)

```suggestion
	return nil, errors.WithStack(herodot.ErrNotFound.WithReason("Unknown namespace with name '" + name"'.))
```